assignments/callbacks.js

```diff
@@ -24,29 +24,71 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x+y);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x*y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
-}
+  let bool = false;
+  for(let i = 0; i<list.length;i++){
+    if(item === list[i]){
+      bool = true;
+    }
+  }
+//   let result = list.reduce(function(isContain, x){
+//     return item === x;
+//   }, false);
+//   return cb(result);
+  return cb(bool);
+ }
+
+// function call(thing){
+//   console.log(thing);
+// }
+// contains('Penci', items, call)
+
+
 
 /* STRETCH PROBLEM */
 
 function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+  
+  // let newArr = [];
+  // for(let i =0; i< array.length; i++){
+  //   if (!(newArr.includes(array[i]))){
+  //     newArr.push(array[i]);
+  //   }
+  // }
+
+  let uniqueArray = array.filter(function(item, pos) {
+    return array.indexOf(item) == pos;
+})
+
+  return cb(uniqueArray);
+}
+
+function consoleL(thing){
+  console.log(thing);
 }
+
+removeDuplicates([1,1,5,3,3,4,4,5], consoleL);
```

Comment:
You could forego the new function here and simply use `console.log` as your callback function:
```javascript
 removeDuplicates([1,1,5,3,3,4,4,5], console.log); 
```
`log` is just method on the `console` object, so you can plug it in as a callback just like any other function!